docs/07-theme-developer-tutorial/01-introduction.md

```diff
@@ -0,0 +1,129 @@
+
+1. CSS
+2. `<head>`
+3. Before Header
+4. After Header
+5. Footer
+6. JS
+
+The "show advanced" toggle can be used to show a few more tabs. Some of those are fairly niche, so we won't go into them for this tutorial. You may also notice the "Mobile" and "Desktop" groups. Those exist for historical reasons, but for new development we recommend implementing everything under 'Common', and using CSS breakpoints for any device-specific tweaks.
+
+### Hello World
+
+Let's start with a basic local theme. We're going to add a big "Hello World!" banner under the Discourse header. In the "After Header" tab, paste this HTML code:
+
+```html
+<div class="welcome-banner">
+  <h1 class="hello-world">Hello World!</h1>
+</div>
+```
+
+And in the CSS tab, add this:
+
+```scss
+.hello-world-banner {
+  height: 300px;
+  width: 100%;
+  background: red;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  margin-bottom: 1em;
+}
+
+.hello-world {
+  font-size: 8em;
+  color: white;
+}
+```
+
+Now hit the save button, then click "Preview". You should see something like:
+
+![12|690x406, 75%](/assets/beginners-guide-12.PNG)
+
+Congratulations! You just created your first Discourse theme! :tada:
+
+### Rendering into an Outlet
+
+"Plugin Outlets" are one of the main ways to customize Discourse. These allow you to add content in thousands of places throughout the user interface. We'll explore these in more detail later, but for now let's render some dynamic content into the `discovery-list-container-top` outlet.
+
+To do this, visit the JS tab of your theme. The code in this tab is a Discourse "API Initializer", and a boilerplate implementation should be filled in for you. To render some content into an outlet, we can use `api.renderInOutlet`, and Ember's `<template>` syntax.
+
+Inside the `apiInitializer`, replace `// your code here` with this:
+
+```gjs
+const currentUser = api.getCurrentUser();
+
+api.renderInOutlet(
+  "discovery-list-container-top",
+  <template>
+    <div class="welcome-banner">
+      {{#if currentUser}}
+        Welcome back @{{currentUser.username}}
+      {{else}}
+        Welcome to our community
+      {{/if}}
+    </div>
+  </template>
+);
+```
+
+Then jump back to the CSS tab, and add this new rule:
+
+```css
+.welcome-banner {
+  background: green;
+  color: white;
+  text-align: center;
+  padding: 10px;
+}
+```
+
+Now save and refresh the preview. You should see a new dynamic welcome banner just above the topic list! Existing users will be welcomed by their username, and new visitors will see a generic message. Nicely done!
+
+There's a lot going on this example: the JS API, Plugin Outlets, Ember's `<template>` tag, and the handlebars template syntax inside that. Don't worry, we'll continue to explore these concepts in more detail throughout the rest of the tutorial.
+
+Next up, we'll demonstrate how to turn this into a "Remote Theme", and take advantage of the extra features they unlock. [Let's go!]()
```

Comment:
Link will be added in a followup PR, because we don't have a topic-id for this doc yet. (🐔 / 🥚 problem)